### ISSUE TITLE
Direct users to tutorial when first opening a project of a new type

### ISSUE THREAD

**Issue by laurentS** [mapswipe/mapswipe#383]
Currently, the app makes it optional to follow the tutorial. With the updated format, we should force the user to go through it when they first try to map a project of a type for they haven't done the tutorial yet.
For experienced users, we can force them through the same way, as a refresher can't hurt, and it's also likely that they don't know all the latest functions of the app (as we've seen through user testing).

This can be implemented with a flag saved in the redux store and persisted to app memory. If the app is uninstalled, the user would have to go through the tutorial again, which might be annoying, but most likely limited in terms of how often it happens (based on the install/uninstall stats of the playstore at least).

@jessica-bergmann91 @jhenshall @chrisbaldock001 happy to have your thoughts on this. Technically, it is easy, and would work exactly like the flag we already have to show first time users the help box when they first open a project.

**Comment by jessica-bergmann91** [comment]
@laurentS YES - I actually think this is essential for the pilot to force users to complete the tutorial BEFORE they can begin any work; and then keep the button for them to come back to the tutorial if they need additional support; I don't mind if they have to complete the tutorial again upon reinstalling ---- it's not like it's a huge time suck
